cli/user.go

```diff
@@ -121,18 +120,36 @@ func runSetUser(cmd *cobra.Command, args []string) {
 		mustUsage(cmd)
 		return
 	}
-	hashed, err := security.PromptForPasswordAndHash()
-	if err != nil {
-		log.Error(err)
-		return
+	var err error
+	var hashed []byte
+	switch password {
+	case "":
+		hashed, err = security.PromptForPasswordAndHash()
+		if err != nil {
+			panic(err)
+		}
+	case "-":
+		scanner := bufio.NewScanner(os.Stdin)
+		scanner.Scan()
```

Comment:
I think we don't need to inspect the return value. However, I noticed that it allows empty password. So I added empty password check below.